INSTALL.md

```diff
@@ -8,47 +8,72 @@ in their docs includes lots of very nice customisations for formatting documenta
 
 ## Building locally
 
+- `git clone --recurse-submodules git@github.com:opensafely/documentation.git`
+  - `--recurse-submodules` ensures you have the necessary copy of the `cohort-extractor` source
 - Create a virtual environment
 - `pip install -r requirements.txt`
 - `mkdocs serve`
 
 ## Updating Cohort Extractor
 
-opensafely-cohort-extractor is a documentation dependency. We use the
-opensafely-cohort-extractor docstrings to generate some content here.
+[cohort-extractor](https://github.com/opensafely-core/cohort-extractor) is a documentation dependency.
+We use the cohort-extractor docstrings to generate some content here.
+
+**We currently do not install cohort-extractor into a virtualenv
+because we are only using the docstrings**
+cohort-extractor is currently a Git submodule instead,
+for installation simplicity (see #832).
```

Comment:
Nit: I don't love the intra-sentence line breaks here.  I know sembr suggests it but it bring me no joy to read! YMMV 🙂

Reply:
I tend to agree, although semantic line breaks have benefits in interpreting subsequent changes and diffs. [^1]

I'll remove them here, because:

* there's a mix of styles
* the likely audience (developers) will likely be reading in a terminal or text editor [^2]

[^1]: Semantic line breaks make Serious Content™ read to the eye like slightly whimsical poetry. After some time, I think I'm getting slowly accustomed to at least *reading* this way, but that takes a while. *Writing* is still more awkward, you either have to:
      * make frequent edits to multiple lines when rewriting
      * write without semantic line breaks until you're done, then add them last.

[^2]: The intended audience is an important question. **Where and how are most people going to read this document?**

      If it's as Markdown rendered into another format, semantic line breaks make more sense. If it's most likely going to be read by someone in a text editor or pager, maybe semantic line breaks are more irritating to have than the benefits to reading subsequent diffs.

Reply:
Done.